glossary.yml

```diff
@@ -67,6 +69,20 @@ entries:
     description: |-
       A cost is properly allocable to a contract if it is incurred specifically for the contract or is incurred for another activity that also benefits the contract. With regard to the allocability of costs claimed, the COR must look for any direct costs that appear to have no connection with the contract work.  If the COR cannot understand why the contractor would need billed materials or effort to perform the contract, then the costs may have been charged to the contract in error.
 
+  American Rescue Plan:
```

Comment:
May want to consider including ARPA, as it is often referred to this way outside of Federal circles. (American Rescue Plan Act).  Also, should legislation carry links? :shrug: https://www.congress.gov/bill/117th-congress/house-bill/1319/text

Reply:
Added ARPA as an acronym for the ARPA. 😜 I am indifferent about linking to the bill text, so if y'all think it should be there, I'll add it; otherwise I won't bother.